app/policies/membership_policy.rb

```diff
@@ -7,7 +7,11 @@ def initialize(person, record)
   end
 
   def index?
-    return true if person.memberships.pluck(:status).include?('member')
+    return true if person.staff == true || person.memberships.pluck(:status).include?('member')
```

Comment:
Not-blocking: [Boolean fields by default give you a predicate method](https://makandracards.com/makandra/607-automagical-methods-for-boolean-attributes-in-activerecord)